Libs/MRML/Core/vtkMRMLSegmentationStorageNode.cxx

```diff
@@ -163,6 +166,7 @@ void vtkMRMLSegmentationStorageNode::InitializeSupportedWriteFileTypes()
     {
     this->SupportedWriteFileTypes->InsertNextValue("Segmentation (.seg.nrrd)");
     this->SupportedWriteFileTypes->InsertNextValue("Segmentation (.nrrd)");
+    this->SupportedWriteFileTypes->InsertNextValue("Segmentation (.nii.gz)");
```

Comment:
Can we write segmentation to nifti format? If we can only write labelmap without segmentation metdata then it is better not to list this format among supported write formats (it is more of a lossy export format then).